src/plugins/data/public/query/query_string/language_service/language_service.ts

```diff
@@ -37,6 +58,87 @@ export class LanguageService {
   public getDefaultLanguage(): LanguageConfig {
     return this.languages.get('kuery') || this.languages.values().next().value;
   }
+
+  public getQueryEditorExtensionMap() {
+    return this.queryEditorExtensionMap;
+  }
+
+  resetUserQuery() {
```

Comment:
i dont think we call this anywhere right we should look to call this when it gets toggled off. in the query enhancements plugin we can listen to the ui settings and reset query if enhancements is false